arc/modal/public_api.py

```
@@ -0,0 +1,93 @@
+        if not MODAL_AVAILABLE:
+            return False
+            
+        # Check if we're configured to use deployed app
+        if not os.environ.get("ARC_USE_DEPLOYED_APP"):
+            return False
+            
+        try:
+            # Try to look up the function
+            modal.Function.lookup("arc-production", "evaluate_single_scenario")
+            return True
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)